pkg/sdk/testint/streams_gen_integration_test.go

```diff
@@ -137,7 +137,7 @@ func TestInt_Streams(t *testing.T) {
 	})
 
 	t.Run("CreateOnDirectoryTable", func(t *testing.T) {
-		stage, cleanupStage := testClientHelper().Stage.CreateStageWithDirectory(t)
+		stage, cleanupStage := testClientHelper().Stage.CreateStageWithDirectory(t, testClientHelper().Ids.SchemaId())
```

Comment:
here is the example where we should just use the method without passing the schema

Reply:
Done.